packages/dds/tree/prettier.config.cjs

```diff
@@ -0,0 +1,42 @@
+// These configurations (excluding js, ts) were taken directly from /workspaces/FluidFramework/packages/dds/tree/prettier.config.cjs
+// TODO: Use shared config in build-common once version with config has been published.
```

Comment:
Are you sure such a version not been published yet?

Reply:
There is a configuration file in `/FluidFramework/common/build/build-common/prettier.config.cjs`, which seems for it to be used across the repo. I have updated the `prettier.config.cjs` file in the `/FluidFramework/packages/dds/tree` directory with this file.